.github/workflows/esm_tools_actions.yml

```diff
@@ -1,62 +1,64 @@
 name: esm-tools main test
 
-# Controls when the action will run. Triggers the workflow on push or pull request.
-
 on: [push, pull_request]
 
 jobs:
   basic_tests:
-    # Containers must run in Linux based operating systems
     runs-on: ubuntu-latest
-    # Docker Hub image that `container-job` executes in
     env:
       LC_ALL: en_US.UTF-8
       LANG: en_US.UTF-8
+
+    strategy:
+      matrix:
+        python-version: ['3.8.10']
+
```

Comment:
If we are bound to the python version of the container we can simply remove this completely. This test is not for testing python versions, we have other tests for that.
```suggestion
```

Reply:
I removed the strategy as suggested. @pgierz can you live with it?

Reply:
(grumble grumble something about beauty and elegance) Yes I can.

Reply:
sorry Paul, we have the same tests without container with all the versions. The advantage of this is that if someone changes the container to a new python version it will still work.